pkgs/tools/networking/trurl/default.nix

```
@@ -0,0 +1,32 @@
+{ lib, stdenv, fetchFromGitHub, runCommand, curl, perl }:
+
+stdenv.mkDerivation rec {
+  pname = "trurl";
+  version = "0.2";
+
+  src = fetchFromGitHub {
+    owner = "curl";
+    repo = pname;
+    rev = "${pname}-${version}";
+    sha256 = "sha256-rUYw5LOHTfjVseODxsW4GhH67SSIpY03l1Gp3ke+xX0=";
+  };
+
+  separateDebugInfo = stdenv.isLinux;
+
+  enableParallelBuilding = true;
+
+  propagatedBuildInputs = [ curl ];
+  makeFlags = [ "PREFIX=$(out)" ];
+
+  doCheck = true;
+  checkInputs = [ perl ];
+  checkTarget = "test";
+
+  meta = with lib; {
+    description = "A command line tool for URL parsing and manipulation.";
```

```suggestion
    description = "A command line tool for URL parsing and manipulation";
```
The description should not end with a period, see
https://nixos.org/manual/nixpkgs/stable/#var-meta-description